backends/files.go

```diff
@@ -188,127 +203,123 @@ func (o *Files) readAcls() (int, error) {
 
 	for scanner.Scan() {
 		index++
-		line := scanner.Text()
 
-		//Check comment or empty line to skip them.
 		if checkCommentOrEmpty(scanner.Text()) {
 			continue
 		}
 
-		//If we see a user line, change the current user.
-		if strings.Contains(line, "user") {
-			//Try to get username
-			lineArr := strings.Fields(line)
+		line := strings.TrimSpace(scanner.Text())
 
-			//Check format
-			if len(lineArr) == 2 && lineArr[0] == "user" {
-				_, ok := o.Users[lineArr[1]]
+		lineArr := strings.Fields(line)
+		prefix := lineArr[0]
 
-				//Check that user exists
-				if !ok {
-					log.Warnf("user %s doesn't exist, skipping acl", lineArr[1])
-					continue
-				}
+		if prefix == "user" {
+			// Since there may be more than one consecutive space in the username, we have to remove the prefix and trim to get the username.
+			username, err := removeAndTrim(prefix, line, index)
+			if err != nil {
+				return 0, err
+			}
 
-				currentUser = lineArr[1]
+			_, ok := o.Users[username]
 
-			} else {
-				return 0, errors.Errorf("Files backend error: wrong acl format at line %d", index)
+			if !ok {
+				log.Warnf("user %s doesn't exist, skipping acls", username)
+				// Flag username to skip topics later.
+				userExists = false
+				continue
 			}
-		} else if strings.Contains(line, "topic") {
 
-			//Split and check for read, write or empty (readwwrite) privileges.
+			userExists = true
+			currentUser = username
+		} else if prefix == "topic" || prefix == "pattern" {
+
 			lineArr := strings.Fields(line)
```

Comment:
This call is unneeded. We can reusing `lineArr` from outside the `if`, it has the same value (and it's reused after the `if`).